cas-server-support-jpa-service-registry/src/main/resources/META-INF/spring/jpa-svc-reg-context.xml

```diff
@@ -35,6 +35,9 @@
        http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd
        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
 
+    <context:component-scan base-package="org.jasig.cas" />
```

Comment:
This package are already scanned: https://github.com/Jasig/cas/blob/master/cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/applicationContext.xml#L42